### ISSUE TITLE
Note fields are being saved with report

### ISSUE THREAD

**Issue by abbyad** [medic/medic#5769]
Note fields in Enketo are being saved with reports as empty fields. Note fields are used as display (as described in [XLSForm documentation](http://xlsform.org/en/#question-types)), and given that they cannot take input they should not be saved with the report. 

**To Reproduce**
1. Submit a report with a notes field.
2. See any note fields saved as empty fields with the report.

**Expected behavior**
I would expect to only see relevant input fields saved with the report, meaning that note fields should not be saved in the report.

**Screenshots**
From https://github.com/medic/medic/issues/5768#issue-462154194 the `"n"` field is the note shown in the screenshot.
```json
  "fields": {
    "start": "2019-06-27T20:00:00.000-04:00",
    "end": "2019-06-27T20:00:00.000-04:00",
    "today": "2019-06-27",
    "deviceid": "deviceid not found",
    "subscriberid": "subscriberid not found",
    "simserial": "simserial not found",
    "phonenumber": "phonenumber not found",
    "username": "username not found",
    "email": "email not found",
    "n": "",
    "meta": {
      "instanceID": "uuid:f31d64b7-19ba-43c1-8e34-657c050c81a3"
    },
    "deprecatedID": ""
  },
```
![image](https://user-images.githubusercontent.com/789512/60363591-f75f7480-99b1-11e9-966c-d4d08003a631.png)

**Environment**
- Instance: gamma.dev and standard.dev
- App: webapp
- Version: 3.6.0 pre-release, and 2.18.0-beta.1

**Comment by garethbowen** [comment]
@abbyad Do you think it would be useful to actually save the note that was shown to the user? This could be hidden from the UI but would potentially be useful for auditing purposes. Obviously storing an empty string every time like we do now is useless.

**Comment by abbyad** [comment]
That could be helpful, as long as it only saves relevant fields. Do you think the bloat from note fields would be significant?

Aside from helping for auditing purposes, the note value could be shown when viewing the submitted form in the History tab- although we may want to figure out how to handle the formatting.

**Comment by garethbowen** [comment]
Yes the bloat would be quite significant. We should only save it if it's likely to be useful.

**Comment by abbyad** [comment]
So the advantages are that note fields would be available for:
1. **Auditing** - I haven't needed this, but perhaps @derickl knows if this would be useful
1. **Report view in History** - tricky to do well in the way we have numerous note fields in the summary page and only show relevant ones, and the formatting wouldn't carry over anyhow.
1. **Testing purposes** - with our test framework we test against outputs of the form, so having what is presented to the user in the summary page (composed entirely of notes) could help. @kennsippell, would that be beneficial, or is there a better way to test what is being presented to the user? 

If these aren't needed, or there are better ways to do all three of these, then the bloat doesn't seem worthwhile. If we find we need it at a later time it seems easier to add it in then, rather than removing it after deployments are expecting it.

**Comment by derickl** [comment]
Auditing would be top on the list if you want a point-in-time snapshot of what was actually shown given that forms evolve quite a bit in the course of a deployment. I think this ties in quite nicely with testing.

**Comment by kennsippell** [comment]
To unit test the content of the summary, we don't need to attach it to the generated report.

**Comment by garethbowen** [comment]
Let's just drop the field for now. I'm not comfortable including the entire note without some investigation into performance issues. Testing can be achieved using the harness. If Auditing would be useful please file a separate feature request so it can be prioritised and evaluated properly.

**Comment by garethbowen** [comment]
Moved to 3.12.0 to free up engineers to work on 3.11.0.
